kalitectl.py

```diff
@@ -592,6 +605,85 @@ def status():
 }
 
 
+def diagnose():
+    """
```

Comment:
I guess we're not running any Django management commands, so `--traceback` won't be helpful anyway. If it fails for some reason, does it display a full traceback (rather than swallowing it)?